packages/vats/src/vat-agoricNames.js

```diff
@@ -0,0 +1,92 @@
+// @ts-check
+// @jessy-check
+import { E, Far } from '@endo/far';
+import { BrandI } from '@agoric/ertp';
+import { provide } from '@agoric/vat-data';
+import { makeDurableZone } from '@agoric/zone/durable.js';
+import { prepareNameHubKit } from './nameHub.js';
+
+/** @param {import('@agoric/zone').Zone} zone */
+const makeBrandStore = zone => {
+  const brandStore = zone.mapStore('Brand');
```

Comment:
Took me a minute to see this wasn't making singletons. I'm still getting used to zones. 

One thing that would help readability is moving `const brandStore` below the kind definition. Kinds should come first. Anything that the kind def depends on should be obvious so it's clear what has to happen in the first crank of restart.

Reply:
right... refactored to just `prepareNatBrand`, without a closure for the state.